client/disputes/new-evidence/index.tsx

```diff
@@ -573,12 +574,7 @@ export default ( { query }: { query: { id: string } } ) => {
 		dispute.status !== 'needs_response' &&
 		dispute.status !== 'warning_needs_response';
 
-	const isVisaComplianceDispute =
-		dispute &&
-		( dispute.reason === 'noncompliant' ||
-			( dispute?.enhanced_eligibility_types || [] ).includes(
-				'visa_compliance'
-			) );
+	const isVisaCompliance = dispute && isVisaComplianceDispute( dispute );
```

Comment:
```suggestion
	const isVisaCompliance = isVisaComplianceDispute( dispute );
```

`isVisaComplianceDispute` will already check for `dispute` to be truthy